Makefile

```diff
@@ -114,9 +114,13 @@ CURRENT_INSTALLER=$(CURRENT_DIR)/installer
 INSTALLER_IMG=$(INSTALLER).$(INSTALLER_IMG_FORMAT)
 INSTALLER_FIRMWARE_DIR=$(INSTALLER)/firmware
 CURRENT_FIRMWARE_DIR=$(CURRENT_INSTALLER)/firmware
-BIOS_IMG=$(INSTALLER_FIRMWARE_DIR)/OVMF.fd $(INSTALLER_FIRMWARE_DIR)/OVMF_CODE.fd $(INSTALLER_FIRMWARE_DIR)/OVMF_VARS.fd
 UBOOT_IMG=$(INSTALLER_FIRMWARE_DIR)/boot
 
+# not every firmware file is used on every architecture
+BIOS_IMG_amd64=$(INSTALLER_FIRMWARE_DIR)/OVMF.fd $(INSTALLER_FIRMWARE_DIR)/OVMF_CODE.fd $(INSTALLER_FIRMWARE_DIR)/OVMF_VARS.fd
```

Comment:
Can you please add BIOS_IMG_riscv64 the same as BIOS_IMG_amd64? Seems we have them in place:
https://github.com/lf-edge/eve/blob/2fe62f18fa90db35d65f484a71e4fa4195305911/pkg/uefi/build.sh#L13-L14

Reply:
Sure, done.